modules/contact/test/tests/3d-mortar-contact/tests

```diff
@@ -10,6 +10,8 @@
     mesh_mode = 'REPLICATED'
     requirement = 'MOOSE shall solve a 3D frictionless bouncing block problem with mortar constraint'
     min_ad_size = 100
+    rel_err = 1.0e-3
+    abs_zero = 1.0e-5
```

Comment:
Are these differences due to the new evaluation of quadrature points for the triangle mesh? If we are sure these new results are better, you could regold?

Reply:
Thanks @recuero ! There are a few additional tests to pass which I'll need to increase the error tolerance. I can regold these if both you and @lindsayad agree to do so.

Reply:
> Are these differences due to the new evaluation of quadrature points for the triangle mesh? If we are sure these new results are better, you could regold?

Yes I would much rather regold if the results are better, than loosen the tolerances. If we just loosen the tolerances then we open ourselves up to regression to a "worse" technique/result in the future